src/db/migrations/20250219182508_add_notifications_table.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.createTable("notifications", (table) => {
+    table.increments("notification_id").primary();
```

Comment:
nit: we usually just use "id" as the incremental primary key for a table

We would use names like "xxx_id" to signal a foreign key (an ID that comes from another table but referenced in this one, ie subscriber_id)